XenOvfApi/OVF.cs

```diff
@@ -4307,13 +4307,15 @@ public static VirtualHardwareSection_Type FindVirtualHardwareSectionByAffinity(E
         #endregion
 
         #region SETs
+
         /// <summary>
-        /// Set the count for the number of CPUs to assign to the virtual machine.
+        /// Set the count for the number of vCPUs and max vCPUs to assign to the virtual machine.
         /// </summary>
         /// <param name="ovfObj">EnvelopeType</param>

```

Comment:
Unrelated to this PR, but since we're here, the parameter name should change from `ovfObj` to `ovfEnv.